discordbot/messages.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, bot : discordbot):
         color = params.pop("color", Colors.get_default(self.bot))
         if isinstance(color, Colors): color = color.value
         e = embeds.build_embed(title=params.pop("title", ""), description=params.pop("message", params.pop("description", "")),
-                               color=params.pop("color", Colors.get_default(self.bot)), **params)
+                               color=color, **params)
```

Comment:
Why do this? The original will grab the color parameter passed, and if not passed use the default color. Yours will grab the color, and if it doesn't exist it can cause error or use discord's default as opposed to bot's default.

Reply:
doesnt pop remove the element from the list? you already did it once at the top, doing it again will always return `Colors.get_default(self.bot)`